eth-contracts/contracts/AudiusAdminUpgradeabilityProxy.sol

```diff
@@ -1,67 +1,73 @@
 pragma solidity ^0.5.0;
 
-import "@openzeppelin/upgrades/contracts/upgradeability/AdminUpgradeabilityProxy.sol";
+import "@openzeppelin/upgrades/contracts/upgradeability/UpgradeabilityProxy.sol";
 
 
 /**
- * Wrapper around OpenZeppelin's AdminUpgradeabilityProxy contract.
+ * Wrapper around OpenZeppelin's UpgradeabilityProxy contract.
  * Permissions proxy upgrade logic to Audius Governance contract.
  * https://github.com/OpenZeppelin/openzeppelin-sdk/blob/release/2.8/packages/lib/contracts/upgradeability/AdminUpgradeabilityProxy.sol
  */
-contract AudiusAdminUpgradeabilityProxy is AdminUpgradeabilityProxy {
+contract AudiusAdminUpgradeabilityProxy is UpgradeabilityProxy {
```

Comment:
should we prefix all these with something? like proxyUpgradeTo, proxyGetAdminAddress, proxyGetImplementationAddress, etc